README.md

```diff
@@ -1,6 +1,7 @@
 # QA Test Automation (QATA) framework
 for UC Berkeley SIS Campus Solutions
 
+[![Build Status](https://travis-ci.org/mDibyo/sis-qa-test-auto.svg?branch=SISRP-10794)](https://travis-ci.org/mDibyo/sis-qa-test-auto)
```

Comment:
@mDibyo yes, thanks - also enabled https://travis-ci.org/ucberkeley/sis-qa-test-auto which should hopefully be active as soon as you do another push.